client/scripts/controllers/chain/substrate/account.ts

```diff
@@ -112,12 +120,27 @@ class SubstrateAccounts implements IAccountsModule<SubstrateCoin, SubstrateAccou
         of(api),
         api.derive.staking.validators(),
         api.query.staking.currentEra(),
+        api.derive.staking.electedInfo()
       )),
 
       // fetch balances alongside validators
-      flatMap(([api, { nextElected, validators: currentSet }, era]: [ApiRx, DeriveStakingValidators, EraIndex]) => {
+      flatMap(([api, { nextElected, validators: currentSet }, era, electedInfo]: [ApiRx, DeriveStakingValidators, EraIndex, DeriveStakingElected]) => {
         // set of not yet but future validators
         const toBeElected = nextElected.filter((v) => !currentSet.includes(v));
+        const validatorsInfo: IValidatorInfo = {};
+
+        electedInfo.info.map(({ accountId, exposure: _exposure, validatorPrefs }): any => {
```

Comment:
This seems a bit off, why is the type for exposure `_exposure`? I believe this can also be a forEach loop instead of a map since we're not using the result.

Reply:
Yes, you are right, I'll update PR.